Library/Formula/qjackctl.rb

```diff
@@ -0,0 +1,135 @@
+class Qjackctl < Formula
+  desc "simple Qt application to control the JACK sound server daemon"
+  homepage "http://qjackctl.sourceforge.net"
+  url "https://downloads.sourceforge.net/qjackctl/qjackctl-0.4.1.tar.gz"
+  sha256 "98024e1ea9f55ac16c290feda051d6494b2261e83f6a918f0fa583b509e2bfd1"
+  head "http://git.code.sf.net/p/qjackctl/code", :using=>:git
+
+  stable do
+    patch :DATA
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "qt"
+  depends_on "jack"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-qt4",
+                          "--disable-dbus",
+                          "--disable-xunique",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+    mv "#{bin}/qjackctl.app", prefix
+    bin.install_symlink "#{prefix}/qjackctl.app/Contents/MacOS/qjackctl"
+  end
+
+  test do
+    Kernel.system "qjackctl", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Also, instead of using `Kernel.system` use `shell_output` with `1` as the second parameter to verify the exit code and verify the command output with `assert_match`